lib/jwt_signed_request/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module JWTSignedRequest
-  VERSION = '2.5.2'.freeze
+  VERSION = '2.5.3'
```

Comment:
`.freeze` is redundant in Ruby 2.3+ because we've already added the magic comment above ⬆️